scripts/add-cluster.sh

```diff
@@ -256,21 +267,31 @@ else
     CLUSTER_JOIN_TO_NAME=`oc get infrastructure cluster -o jsonpath='{.status.infrastructureName}' ${OC_ADDITIONAL_PARAMS}`
 fi
 
+echo "Creating ${JOINING_CLUSTER_TYPE} secret"
 SECRET_NAME=${SA_NAME}-${JOINING_CLUSTER_NAME}
 if [[ -n `oc get secret -n ${CLUSTER_JOIN_TO_OPERATOR_NS} ${OC_ADDITIONAL_PARAMS} | grep ${SECRET_NAME}` ]]; then
     oc delete secret ${SECRET_NAME} -n ${CLUSTER_JOIN_TO_OPERATOR_NS} ${OC_ADDITIONAL_PARAMS}
 fi
 oc create secret generic ${SECRET_NAME} --from-literal=token="${SA_TOKEN}" --from-literal=ca.crt="${SA_CA_CRT}" -n ${CLUSTER_JOIN_TO_OPERATOR_NS} ${OC_ADDITIONAL_PARAMS}
 
+if [[ ${MULTI_NS} != "true" ]]; then
+    TOOLCHAINCLUSTER_NAME=${JOINING_CLUSTER_TYPE_NAME}-${JOINING_CLUSTER_NAME}
+    OWNER_CLUSTER_NAME=${CLUSTER_JOIN_TO}-${CLUSTER_JOIN_TO_NAME}
+else
+    echo "Multi-namespace member mode"
+    TOOLCHAINCLUSTER_NAME=${JOINING_CLUSTER_TYPE_NAME}-${JOINING_CLUSTER_NAME}-2
+    OWNER_CLUSTER_NAME=${CLUSTER_JOIN_TO}-${CLUSTER_JOIN_TO_NAME}-2
```

Comment:
btw, just a small comment:
instead of introducing a boolean param `--multi-ns` you could add for example `--suffix-name` parameter that would add the suffix after both the `ToolchainCluster` CR as well as the ServiceAccount name.
We could call:
```
add-cluster.sh --suffix-name "-2"
```
so it would create SA `toolchaincluster-member-2` or `e2e-service-account-2`
analogically also the ToolchainCluster CRs.

It would also simplify the logic since you could only append the variable:
```
 ${JOINING_CLUSTER_TYPE_NAME}-${JOINING_CLUSTER_NAME}${SUFFIX_NAME}
```
without introducing the if-else statements

This is a proposal for improvement, it's nothing I would insist on nor require to be implemented in this PR ;-)

Reply:
That's a great idea! I'll do that.